net/datacenter-node-install/setup-dc-node-1.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+if [[ "$EUID" -ne 0 ]]; then
+  echo "Please run as root"
+  exit 1
+fi
+
+if [[ -n "$1" ]]; then
+  PUBKEY_FILE="$1"
+else
+  cat <<EOF
+Usage: $0 [pubkey_file]
+
+The pubkey_file should be the pubkey that will eb set up to allow the current user
+(assumed to be the machine admin) to log in via ssh
+EOF
+  exit 1
+fi
+
+USERNAME="$(whoami)"
+
+set -xe
+
+apt update
+apt upgrade -y
+apt install -y build-essential pkg-config clang
+
+../scripts/install-docker.sh
```

Comment:
Instead of relative paths, these should all be relative to the location of the running script, otherwise running them from some other dir other than what is assumed here will not work.